Python/Tests/Analysis/LanguageServerTests.cs

```diff
@@ -632,19 +633,19 @@ class C:
         [TestMethod, Priority(0)]
         public async Task HoverSpanCheck() {
             var s = await CreateServer();
-                var mod = await AddModule(s, @"import datetime
+            var mod = await AddModule(s, @"import datetime
 datetime.datetime.now().day
 ");
 
             await AssertHover(s, mod, new SourceLocation(2, 1), "built-in module datetime*", new[] { "datetime" }, new SourceSpan(2, 1, 2, 9));
-            if (Default.Version < Microsoft.PythonTools.Parsing.PythonLanguageVersion.V30) {
+            if (Default.Version.Is2x()) {
```

Comment:
We should check the type of the test class (`this is LanguageServerTests_V2`) rather than the default factory.